parts/k8s/addons/1.10/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -395,7 +390,7 @@ spec:
         # This container installs the Calico CNI binaries
         # and CNI network config file on each node.
         - name: install-cni
-          image: quay.io/calico/cni:v3.1.3
+          image: quay.io/calico/cni:v3.4.0-0.dev-13-gcda66d6
```

Comment:
Is there a final version for the cni image, say the equivalent of `quay.io/calico/typha:v3.3.0`?